shared/main.desktop.js

```diff
@@ -18,9 +21,16 @@ type Props = {
   routeState: RouteStateNode,
   setRouteState: (path: Path, partialState: {}) => void,
   checkReachability: () => void,
+  getUserImage: () => void,
 }
 
 class Main extends Component<void, Props, void> {
+  constructor (props: Props) {
+    super(props)
+
+    initAvatarLookup(getUserImage)
```

Comment:
This doesn't need to be connected to the redux stores and it doesn't depend on the main component, so it should be initialized with the other globals in https://github.com/keybase/client/blob/master/desktop/renderer/index.js